js/src/shell/index.js

```diff
@@ -85,13 +85,12 @@ function onEnterDapp ({ params }) {
 
 ReactDOM.render(
   <ContextProvider api={ api } muiTheme={ muiTheme } store={ store }>
-    <Router history={ hashHistory }>
-      <Route path='/' component={ Application }>
-        <Redirect from='/auth' to='/' />
+    <Router>
+      <Application>
+        <Route path='/' exact component={ Dapps } />
```

Comment:
We can't lose the /auth route, used by the shell to generate and specify new tokens.